lib/internal/Magento/Framework/View/Page/Config/Renderer.php

```diff
@@ -131,21 +133,35 @@ public function renderMetadata()
     /**
      * @param string $name
      * @param string $content
-     * @return mixed
+     * @return string
      */
     protected function processMetadataContent($name, $content)
     {
-        $method = 'get' . $this->string->upperCaseWords($name, '_', '');
-        if ($name === 'title') {
```

Comment:
Looks like additional logic for title attribute should be here, please, revert.

Reply:
@sidolov Do you want to keep old logic for 2.3 ?

Reply:
@gelanivishal , yes, this logic should be here due to bugfix